docs/files.md

```diff
@@ -746,3 +747,24 @@ A file's watermark can be removed by calling
 ```js
 client.files.removeWatermark('67890', callback);
 ```
+
+Get Representation Info
+-----------------------
+
+A file's representation info can be retrieved by calling
+[`files.getRepresentationInfo(fileID, representationTypes
+callback)`](https://opensource.box.com/box-node-sdk/Files.html#getRepresentationInfo).
+You will be able to fetch information regarding pdf representation, thumbnail representation, multi-page images
+representation, and extracted text representation.
+
+You can retrieve information regarding the generated representations by calling. This will retrieve information
+for a 2048x2048 jpg representation and a 2048x2048 png representation generated for your Box file.
+```js
+client.files.getRepresentationInfo('67890', client.files.representation.IMAGE_LARGE, callback);
+```
+
+Similarly you can form your own request by manually passing in the representation types you want to
```

Comment:
It might be good to link to the API documentation (https://developer.box.com/reference#section-x-rep-hints-header) here so users know what the format of this should look like.